movie_backend/movie_api/views.py

```diff
@@ -0,0 +1,60 @@
+from django.shortcuts import render
+
+from .models import Movie
+from .serializers import MovieSerializer
+
+from django.http import Http404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+
+# Create your views here.
+
+class MovieList(APIView):
+    '''
+    List all movie, or create a new movie
+    '''
+    serializer_class = MovieSerializer
+    def get(self, request):
```

Comment:
it would be nice to describe the class method, here is an example:
```
"""
		This methods creates a new job category instance
		args:
		request(dictionary)-the data attribute of the request should contain:
			- category_name(string)- the name of the job category
			- description(string)- the description of the category
			- image(string)- the image to represent the category
			
		- Returns Response with a record of the job instance created 
		- Returns Status of the API call = (201 -> created), (400 -> Bad request)
	"""
```